capella2polarion/converters/link_converter.py

```diff
@@ -201,12 +208,13 @@ def _create(
 
     def _handle_exchanges(
         self,
-        obj: fa.Function,
+        obj: common.GenericElement | diag.Diagram,
         work_item_id: str,
         role_id: str,
         attr_id: str,
         links: dict[str, polarion_api.WorkItemLink],
     ) -> list[polarion_api.WorkItemLink]:
+        assert isinstance(obj, common.GenericElement)
         exchanges: list[str] = []
         objs = _resolve_attribute(obj, attr_id)
         exs = self._get_work_item_ids(work_item_id, objs.by_uuid, role_id)
```

Comment:
Do we need this method anymore? I think it is all handled by the new `_resolve_attribute` function